File: C#/Loksabha.cs

class LokSabha {
    private Speaker speaker;
    private MemberOfParliament[] members;
    private int counter = 0;

    public LokSabha(Speaker speaker){
        Console.WriteLine("Lok Sabha created....");
        this.speaker = speaker;
        members = new MemberOfParliament[543];
    }

    public void electLeaders() {
        Console.WriteLine("Lok Sabah is electing members...");
    }

    public void passBill(Bill bill) {
        Console.WriteLine("Lok Sabah is passing bill :\n"+bill.getBill());
    }

    public void addMember(MemberOfParliament member){
        if(counter<543){   
            members[counter] = member;
            counter++;
            Console.WriteLine(members[counter-1].getMPDetails()+ " added to Lok Sabha...");
        }
        else{
            Console.WriteLine("Lok Sabha is full...");
        }
    }

    public string getSpeaker(){
        string sp = this.speaker.getMPDetails();
        return sp;
    } 

    public MemberOfParliament[] getMPsLokSabha(){
        return this.members;
    }
}

File: PY/loksabha.py
# lok Sabha

class LokSabha:
    def __init__(self, speaker):
        print("Lok Sabha created..\n")
        self.speaker = speaker
        self.members = []
        
    def pass_bill(self,bill):
        print(f"Bill {bill.title} passed by the Lok Sabha.\n")
        
    def add_members(self, member):
        if(len(self.members)<543):
            self.members.append(member)
            print(f"{member.get_mp_details()} added to Lok Sabha..\n")
        else:
            print("Lok Sabha full..\n")
            
    def get_speaker(self):
        return self.speaker.get_mp_details()
    
    def get_mps_lok_sabha(self):
        return self.members